parser/css.go

```diff
@@ -0,0 +1,64 @@
+package parser
+
+import (
+	"os"
+	p "path"
+	"path/filepath"
+	"text/template"
+
+	"github.com/spf13/hugo/helpers"
+	jww "github.com/spf13/jwalterweatherman"
+)
+
+// parse parses the file of the given path as templates,
+// injects variables of the config safes the modified file.
+func parse(path string) {
+	t, err := template.ParseFiles(path)
+
+	if err != nil {
+		jww.WARN.Println(err)
+		return
+	}
+
+	// Overwrite the original stylesheet
+	f, err := os.Create(path)
+
+	if err != nil {
+		jww.WARN.Println("Can't overwrite the CSS file: ", err)
+		return
+	}
+
+	err = t.Execute(f, nil)
+	if err != nil {
+		jww.WARN.Println("Can't interpret the CSS file: ", err)
+		return
+	}
+
+	f.Close()
```

Comment:
Do a `defer f.close()` right after you open it, to make sure it gets closed if you return earlier.